packages/game-server/src/shared/events.ts

```diff
@@ -9,6 +9,8 @@ export const Events = {
   START_CRAFTING: "startCrafting",
   STOP_CRAFTING: "stopCrafting",
   PLAYER_DEATH: "playerDeath",
+  SCATTER_LOOT: "scatterLoot",
```

Comment:
where is this used again?

Reply:
when you loot zombie, I put it here but better to have different "internal" set of events

Reply:
yeah I think having 3 separate ways to track events

ServerSentEvents
ClientSentEvents
InternalServerEvents